appveyor.yml

```diff
@@ -100,9 +117,13 @@ test_script:
   - "%CMD_IN_ENV% python setup.py develop"
   - "python -m obspy.scripts.runtests --no-flake8 -n appveyor-ci -r %CI_URL% %PR_URL%"
 
-matrix:
-  # enable fast fail strategy
-  fast_finish: true
+
+on_success:
+  - ps: |
+      If ($env:APPVEYOR_REPO_TAG -eq "true") {
+          python setup.py sdist bdist_wheel
```

Comment:
any thoughts on the comment above - or do you really dislike egg/exe file that much so that you dont even bother to comment on it ? ^^